app/services/ui/host_description.rb

```diff
@@ -16,4 +16,20 @@ def self.reduce_providers(customization_point)
       end
     end
   end
+
+  class HostGroupDescription
```

Comment:
nitpick, we're using `Hostgroup` as a single word, as opposed to `HostGroup`
```suggestion
  class HostgroupDescription
```